src/components/organisms/CesiumViewer.vue

```diff
@@ -209,100 +206,13 @@ export default defineComponent({
-        let activeShape: number | null;
-        let floatingPoint: Entity | null;
-        const handler = new Cesium.ScreenSpaceEventHandler(cesiumViewer.value.canvas);
-        handler.setInputAction((event: { position: number }) => {
-          const earthPosition = cesiumViewer.value.camera.pickEllipsoid(event.position);
-          if (Cesium.defined(earthPosition)) {
-            if (activeShapePoints.length === 0) {
-              floatingPoint = createPoint(earthPosition);
-              activeShapePoints.push(earthPosition);
-              // eslint-disable-next-line max-len
-              const dynamicPositions = new Cesium.CallbackProperty((() => new Cesium.PolygonHierarchy(activeShapePoints)), false);
-              activeShape = drawShape(dynamicPositions);
-            }
-            activeShapePoints.push(earthPosition);
-            createPoint(earthPosition);
-          }
-        }, Cesium.ScreenSpaceEventType.LEFT_CLICK);
-        handler.setInputAction((event: { endPosition: number }) => {
-          if (Cesium.defined(floatingPoint)) {
-            const newPosition = cesiumViewer.value.camera.pickEllipsoid(event.endPosition);
-            if (floatingPoint?.position && Cesium.defined(newPosition)) {
-              (floatingPoint.position as ConstantPositionProperty).setValue(newPosition);
-              activeShapePoints.pop();
-              activeShapePoints.push(newPosition);
-            }
-          }
-        }, Cesium.ScreenSpaceEventType.MOUSE_MOVE);
-        const terminateShape = () => {
-          activeShapePoints.pop();
-          drawShape(activeShapePoints);
-          cesiumViewer.value.entities.remove(floatingPoint);
-          cesiumViewer.value.entities.remove(activeShape);
-          floatingPoint = null;
-          activeShape = null;
-          activeShapePoints = [];
-          useMap.value = false;
-        };
-        handler.setInputAction(() => {
-          activeShapePoints.forEach((element) => {
-            polyPoints[0].push([
-              Cesium.Math.toDegrees(
-                (Cesium.Cartographic.fromCartesian(element)
-                ).longitude,
-              ),
-              Cesium.Math.toDegrees(
-                (Cesium.Cartographic.fromCartesian(element)
-                ).latitude,
-              ),
-            ]);
-          });
-          polyPoints[0].push(polyPoints[0][0]);
-          drawnShape.value.type = 'Polygon';
-          drawnShape.value.coordinates = polyPoints;
-          terminateShape();
-        }, Cesium.ScreenSpaceEventType.LEFT_DOUBLE_CLICK);
-      }
+      Cesium.Camera.DEFAULT_VIEW_FACTOR = 0;
     });
 
     watch(specifiedShape, () => {
-      const uploadedFootprint: number[] = [];
-      specifiedShape.value.coordinates[0].forEach((e: number[]) => {
+      const uploadedFootprint: Cartesian3[] = [];
+      // eslint-disable-next-line  @typescript-eslint/no-explicit-any
+      specifiedShape.value.coordinates.forEach((e: any) => {
```

Comment:
Seems like we can just do

```typescript
const uploadedFootprint: Cartesian3[] = specifiedShape.value.coordinates.map(
   ...
```

Reply:
Getting an error "this expression is not callable" on the map statement?

I attempted it like this:
```
const uploadedFootprint: Cartesian3[] = specifiedShape.value.coordinates.map(
        (e: any) => Cesium.Cartesian3.fromDegrees(e[0], e[1]),
);
```

Reply:
Is `coordinates` an Array? I assumed if it was valid to call `forEach` on it, then `map` would work too.

Reply:
`coordinates` is typed as `number[][][] | number [][][][]`. I thought map would work on it as well, but it gives the following compile error: 

```
Each member of the union type '(<U>(callbackfn: (value: number[][], index: number, array: number[][][]) => U, thisArg?: any) => U[]) | (<U>(callbackfn: (value: number[][][], index: number, array: number[][][][]) => U, thisArg?: any) => U[])' has signatures, but none of those signatures are compatible with each other.
```

Reply:
I changed this to use `Cesium.GeoJsonDataSource` to create the `Entity` from our `Polygon` in e757ee6